### ISSUE TITLE
ENError 1 

### ISSUE THREAD

**Issue by ndegendogo** [corona-warn-app/cwa-app-ios#4356]
## Avoid duplicates
* [ ] Question is not already answered in the [FAQ](https://www.coronawarn.app/en/faq/)
* [ ] Question is specific for iOS only, for general issues / questions that apply to iOS and Android please raise them in the [documentation repository](https://github.com/corona-warn-app/cwa-documentation)
* [ ] Question has not already been asked in another issue

## Your Question

* Source File:
* Line(s):
* Question:

Today I returned home from the test center and opened cwa because I wanted to scan my QR code. The dialog was not very responsive, I had to press all the buttons several time. Note: in parallel it performed the risk check, but I noticed that only later.
And then I got the error message, see screenshot.
After that I could finish the dialog to register my QR code and request my test result and certificate.
Note: both timestamps in cwa and ENF log show this risk check at 11:03 - so was it then successfull? What does ENError 1 mean?

iPhone 13 Pro / iOS 15.1 / cwa 2.19.0 (4) / CCL 1.1.0

---
Internal Tracking ID: [EXPOSUREAPP-12371](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12371)

**Comment by ndegendogo** [comment]
![9FA27882-F1B6-4580-BD18-7981555C0405](https://user-images.githubusercontent.com/6118267/159158138-98d064b7-fec2-4bcf-acfa-518e6c6eac2a.png)

**Comment by thomasaugsten** [comment]
EN Error 1 is from the Apple documentation UNKNOWN
https://developer.apple.com/documentation/exposurenotification/enerror/code/unknown

**Comment by Ein-Tim** [comment]
Twitter Reports reg. this:
- https://twitter.com/dermoritz14/status/1501930962942234626?s=21
- https://twitter.com/bembeltogo/status/1493198254355492867?s=21

I also encountered this bug more than one time. It seems like this happens in combination with the unresponsive behaviour.

**Comment by ndegendogo** [comment]
> EN Error 1 is from the Apple documentation UNKNOWN
@thomasaugsten thanks - so we may assume not even Apple knows what was happening here? 😅

**Comment by dsarkar** [comment]
@ndegendogo @Ein-Tim 

The dev team looked into this issue, and apparently this is something not solvable from development side. Either this  needs a fix from Apple or it could be that it is already fixed.

Unless there are no more reports on this topic this issue considered obsolete for now. Thanks for reporting!

Best wishes, DS

---
Corona-Warn-App Open Source Team

**Comment by ndegendogo** [comment]
I have not seen this error code again.
And I agree that the app development team cannot further analyse or solve this.

Closing.
